.github/templates/global_l0/genesis/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
     - name: Generate Global L0 JAR
       shell: bash
       run: |
-        sbt dagL0/assembly keytool/assembly wallet/assembly
+        sbt -Dversion.override=99.99.99 dagL0/assembly keytool/assembly wallet/assembly
```

Comment:
You can remove the 
-Dversion.override=99.99.99
it's unnecessary here

Reply:
:+1: